docs/sql-guide/functions/function-totimestamp.md

```diff
@@ -0,0 +1,78 @@
+---
+title: TOTIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# TOTIMESTAMP() function
+
+The `TOTIMESTAMP()` function converts the given integer value to a timestamp and returns it. It uses the [unix epoch](https://www.unixtutorial.org/unix-epoch/) as the base and calculates date-time equivalent of the integer.
+
+## Syntax
+
+```
+TOTIMESTAMP(int_exp, timeunit)
+```
+
+## Arguments
+
+
+| Argument | Description | Data type |
```

Comment:
Add a column between Description and Data type so we can make it clearer what's optional and what's not

```
| Argument | Data type | Description | Required |
```